api/ark/backups/get.go

```diff
@@ -0,0 +1,43 @@
+// Copyright © 2018 Banzai Cloud
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package backups
+
+import (
+	"net/http"
+
+	"github.com/gin-gonic/gin"
+	"github.com/pkg/errors"
+
+	"github.com/banzaicloud/pipeline/api/ark/common"
+	"github.com/banzaicloud/pipeline/internal/platform/gin/correlationid"
+)
+
+// Get gets an ARK backup
+func Get(c *gin.Context) {
+	backupName := c.Param("name")
+
+	logger := correlationid.Logger(common.Log, c).WithField("backup", backupName)
```

Comment:
Are you sure we want to gettig backups by `name` instead of `id`?

Reply:
You're right. I'll change it to `id` for the backup/bucket/restore related requests.

I would probably stick with the `name` for the schedules. They are only exists within the cluster and the name is unique there.

Reply:
OK, thx